Stripe/NSString+Stripe.m

```diff
@@ -28,6 +35,29 @@ - (NSString *)stp_reversedString {
     return [mutableReversedString copy];
 }
 
+- (NSString *)stp_stringByRemovingCharactersFromSet:(NSCharacterSet *)cs {
```

Comment:
yeahh – backstory here is that a user optimized a much simpler implementation: https://github.com/stripe/stripe-ios/pull/426